app.py

```diff
@@ -1,13 +1,19 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import requests
+import os
+import trelloapp
+import json
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
```

Comment:
This line (and the `flask_config.py` file) is no longer needed (as we should longer be using the flask session state)

Reply:
Same for `session_items.py` and the SECRET_KEY env variable (both can go)